build.psm1

```diff
@@ -1198,18 +1198,23 @@ function Start-PSPackage {
 
     # Build the name suffix for win-plat packages
     if ($IsWindows) {
-        $VersionTokens = $Version -split "-"
-        if ($VersionTokens.Count -gt 1) {
-            # Get the suffix like 'alpha.10-win81-x64'
-            $NameSuffix = $VersionTokens[1], $Runtime -join "-"
+        # Add the server name to the $RunTime. $runtime produced by dotnet is same for client or server
+        switch ($Runtime) {
+            'win81-x64' {$NameSuffix = 'win81-win2k12r2-x64'}
+            'win10-x64' {$NameSuffix = 'win10-win2k16-x64'}
+            'win7-x64'  {$NameSuffix = 'win7-win2k8r2-x64'}
```

Comment:
Given, there is no x86 version of w2K8r2, win7-x86 case should be covered by the `Default {$NameSuffix = $Runtime}` below, right?